Lib/urllib/response.py

```diff
@@ -73,6 +73,10 @@ def __init__(self, fp, headers, url, code=None):
         self.url = url
         self.code = code
 
+    @property
+    def status(self):
```

Comment:
Why add status?

Reply:
See my comment under "Should status or code be the obvious recommended attribute?"